examples/swapAndBridge.ts

```diff
@@ -71,6 +81,47 @@ const swapAndBridge: Parameters<typeof withAnvilProvider>[0] = async (
     buyTokenBalance: OrderBalance.ERC20,
   };
 
+  // weiroll required since swap output amount is not known
+  // at tx time. need to fetch it at execution time.
+  const weirollCommands = [
+    // balanceOf staticcall
+    encodeCommand(
+      fnSelector('balanceOf(address)'),
+      encodeFlag(false, false, CallType.StaticCall),
+      encodeInput(
+        encodeInputArg(true, 0),
```

Comment:
nit: describe what this is doing. Write it long hand in comments. Developers should be able to reference this example and have minimum additional work required to understand what is happening so they can then apply it to their own use case.